release_bot/webhooks.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+This module is backend for WSGI.
+"""
+from flask import request, jsonify
+from flask.views import View
+
+from release_bot.exceptions import ReleaseException
+
+
+class GithubWebhooksHandler(View):
+    """
+        Handler for github callbacks.
+    """
+
+    def __init__(self, release_bot, conf):
+        self.release_bot = release_bot
+        self.conf = conf
+        self.logger = conf.logger
+
+    def dispatch_request(self):
+        self.logger.info(f'New github webhook call from '
+                         f'{self.conf.repository_owner}/{self.conf.repository_name}')
+        if request.is_json:
+            self.parse_payload(request.get_json())
+        else:
+            self.logger.error(f'This webhook doesn\'t contain JSON')
+        return jsonify(result={"status": 200})
+
+    def parse_payload(self, webhook_payload):
```

Comment:
I thought this function would pass parameters directly to the `handle_*` methods so they can process them right away, instead of them trying to figure things out. But that can be addressed in a separate PR since there would be quite a few changes needed to achieve that.